messaging/src/main/java/org/axonframework/messaging/eventhandling/SimpleEventHandlingComponent.java

```diff
@@ -40,31 +43,51 @@
  * @author Steven van Beelen
  * @since 5.0.0
  */
-public class SimpleEventHandlingComponent implements EventHandlingComponent {
+public class SimpleEventHandlingComponent implements
+        EventHandlingComponent,
+        DescribableComponent {
 
     private static final SequencingPolicy DEFAULT_SEQUENCING_POLICY = new HierarchicalSequencingPolicy(
             SequentialPerAggregatePolicy.instance(),
             SequentialPolicy.INSTANCE
     );
 
+    private final String name;
     private final ConcurrentHashMap<QualifiedName, List<EventHandler>> eventHandlers = new ConcurrentHashMap<>();
     private final SequencingPolicy sequencingPolicy;
 
     /**
-     * Initializes a {@code SimpleEventHandlingComponent} with no {@link EventHandler}s and default
-     * {@link SequentialPolicy}.
+     * Instantiates a simple {@link EventHandlingComponent} that is able to handle events and delegate them to
+     * subcomponents.
+     * <p>
+     * Uses a default {@link SequentialPolicy} that will first try for the {@link SequentialPerAggregatePolicy}, falling
```

Comment:
Should this not be "uses a default sequencing policy" ?
```suggestion
     * Uses a default sequencing policy that will first try for the {@link SequentialPerAggregatePolicy}, falling
```